sql_metadb/derived_tables/feesfines_accounts_actions.sql

```diff
@@ -1,44 +1,96 @@
 --metadb:table feesfines_accounts_actions	
-
--- Create a derived table that takes feesfines_accounts as the main table
--- join all transaction data from the feesfines_actions table
--- add patron group information from user_group table
+--Creates a derived table that takes folio_feesfines.accounts as the main table
+--Payments and credits are shown in as a negative balance for accounting purposes
 
 DROP TABLE IF EXISTS feesfines_accounts_actions;
 
 CREATE TABLE feesfines_accounts_actions AS
+
```

Comment:
Could we remove blank lines in the middle of a query?  Sorry that is not documented anywhere.